packages/storage-s3/src/index.ts

```diff
@@ -39,7 +42,15 @@ export type S3StorageOptions = {
   /**
    * Collection options to apply the S3 adapter to.
    */
-  collections: Partial<Record<UploadCollectionSlug, Omit<CollectionOptions, 'adapter'> | true>>
+  collections: Partial<
+    Record<
+      UploadCollectionSlug,
+      | ({
+          signedDownloads?: SignedDownloadsConfig
```

Comment:
I would prefer this to be moved to the top level property on the storage adapter since that is what we already do for `clientUploads`.

If you wanted the property on the collection like you have here, then we can use this property to override the setting for a specific collection.